src/components/Event/index.js

```diff
@@ -138,34 +138,39 @@ const Event = ({ match, location }) => {
                                 </Alert>
                             }
                             <form onSubmit={handleSubmit} noValidate autoComplete="off">
-                                <div style={{ marginBottom: '40px', display: 'flex', flexDirection: 'column' }}>
+                                <div style={{ marginBottom: '40', display: 'flex', flexDirection: 'column' }}>
```

Comment:
Should be integer values to have the default `px` unit (same for other places in this PR)
```suggestion
                                <div style={{ marginBottom: 40, display: 'flex', flexDirection: 'column' }}>
```